_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -57,6 +57,23 @@ default cmake.build_dir             {${workpath}/build}
 # minimal/initial value for the install rpath:
 default cmake.install_rpath         ${prefix}/lib
 
+# CMake provides several different generators corresponding to different utities
+# (and IDEs) used for building the sources. We support "Unix Makefiles" (the default)
+# and Ninja, a leaner-and-meaner alternative.
+#
+# In the Portfile, use
+#
+# cmake.generator Ninja
+# or
+# cmake.generator Unix Makefiles    # no quotes!
+#
+default cmake.generator             {"Unix Makefiles"}
+# CMake generates Unix Makefiles that contain a special "fast" install target
+# which skips the whole "let's see if there's anything left to (re)build before
+# we install" you normally get with `make install`. That check should be
+# redundant in normal destroot steps, because we just completed the build step.
+default destroot.target             install/fast
```

Comment:
OK for the curly braces, but how does that relate to the remark you made about the `default cmake.generator` expression?